frontend/src/components/abfragevarianten/SachbearbeitungComponent.vue

```diff
@@ -26,6 +26,8 @@
           cols="12"
           md="6"
         >
+        </v-col>
+        <!-- Das Datum wird in ISI 2.0 wieder relevant (siehe ISI-1238)
```

Comment:
Da der Code in github verfügbar ist und unser internes Jira nicht aus dem Internet erreichbar ist, sollte der Verteil auf ISI-1238 im Kommentar entfernt werden.